mmdet3d/core/utils/gaussian.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
```

Comment:
These three functions in this file seem to have duplicated implementations in [mmdet](https://github.com/open-mmlab/mmdetection/blob/046db20e25636d6334ca4d2c8180a838afb11fca/mmdet/models/utils/gaussian_target.py) . We can directly use them in mmdet.